install/gcp/deployment_manager/istio-cluster.yaml

```diff
@@ -9,12 +9,15 @@ resources:
     gkeClusterName: istio-cluster
     zone: us-central1-a
     initialNodeCount: 4
-    instanceType: n1-standard-1
+    instanceType: n1-standard-2
     enableAutomaticSidecarInjection: true
     enableMutualTLS: true
     enablePrometheus: true
     enableGrafana: true
     enableTracing: true
-    enableServiceGraph: true
+    enableServiceGraph: false
+    enableKiali: true
+    kialiUsername: admin
```

Comment:
zug - can you fix this so there is atleast no default password - and one must be specified?

Reply:
I can remove the default password and add specifications for
`minLength: 8`
and some regex pattern so that users have to specify a password.

_however_, the DM GUI doens't do any presubmit checks.  Which means the user will see a blank entry for `kialiPassword` and still allowed to click continue...the DM script will fail on the next step though...Its pretty awkward, unfortuantely.

Reply:
neverminds. its actually there in the `.schema` where you define required fields.   I set `kialiPassphrase` to required and set its `minLength: 8`  

![ss](https://user-images.githubusercontent.com/11149054/50752846-772b3400-1204-11e9-9941-cb1938835206.png)

i'll update this on the next push after addressing the ports used